pkg/auth/jwt.go

```diff
@@ -110,19 +110,23 @@ func (j *JwtAuthenticator) ParseAndValidate(tokenString string) (jwt.MapClaims,
 // The keys are in a public key format and are converted to RSA Public Keys
 func (j *JwtAuthenticator) refreshJwksKeys() error {
 	oidcURL := os.Getenv(OIDCServerURL)
-
+	fmt.Println("***********InSideAuthJwt_SMAL_************")
+	fmt.Sprintf("oidcURL :-%s\n",oidcURL)
```

Comment:
can you give some context for why you're adding this? Like a link to some documentation about SAML? Will this work in a non SAML situation?
You could just have added it the the Sprintf on the line below